qcut/docs/completed/bug_v4.md

```
@@ -0,0 +1,156 @@
+index-D01df2Qo.js:444 ⚠️ RAPID UPDATES DETECTED! Object
+pn @ index-D01df2Qo.js:444
+index-D01df2Qo.js:444 🔍 [PanelStore] [15] setPropertiesPanel:SKIP +0ms Size unchanged
+index-D01df2Qo.js:444 ⚠️ RAPID UPDATES DETECTED! Object
+pn @ index-D01df2Qo.js:444
+index-D01df2Qo.js:444 🛡️ [TOLERANCE-FIX] setPropertiesPanel BLOCKED: 33.33 vs 33.33 (diff: 0 < 0.1)
+index-D01df2Qo.js:563 🎯 [EditorPage] Render #2, Project ID: b93cadda-e6a5-4bab-9770-079bfe98fb93
+index-D01df2Qo.js:563 🎯 [EditorPage] Panel sizes: Object
+index-D01df2Qo.js:563 🎯 [EditorPage] Render #3, Project ID: b93cadda-e6a5-4bab-9770-079bfe98fb93
+index-D01df2Qo.js:563 🎯 [EditorPage] Panel sizes: Object
+vendor-CvAI8bIM.js:138 Uncaught Error: Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.
+    at L_ (vendor-CvAI8bIM.js:155:759)
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)